lib/Driver/ToolChains.cpp

```diff
@@ -334,6 +334,8 @@ void ToolChain::addCommonFrontendArgs(const OutputInfo &OI,
     auto OptArg = inputArgs.getLastArgNoClaim(options::OPT_O_Group);
     if (!OptArg || OptArg->getOption().matches(options::OPT_Onone))
       arguments.push_back("-enable-anonymous-context-mangled-names");
+
+    inputArgs.AddAllArgs(arguments, options::OPT_debug_compilation_dir);
```

Comment:
Does this code need to be added to the Swift implementation of the driver, too? (I haven't looked at the Swift driver code recently so I'm not sure)

Reply:
Yes, I think so. My plan is to address this in the follow up PR.

Reply:
I think it may be preferable to do both at once, to keep the implementations in sync at all times.

Reply:
Create another [PR](https://github.com/apple/swift-driver/pull/968) for Swift-implemented driver.